src/components/Menu/__stories__/Default.tsx

```diff
@@ -0,0 +1,24 @@
+import { fireEvent, screen } from '@storybook/testing-library'
+import Menu from '..'
+import { Template } from './Template'
+
+export const Default = Template.bind({})
```

Comment:
Optional: We could call it "Playground"

Reply:
Naming it `Playground` definitely makes more sense 👌

Reply:
Strongly agree!

Reply:
Sounds perfect !